File: kotlin/src/atcoder/tokubeki10mon/01_ABC086A.kt
package atcoder.tokubeki10mon

fun main(args: Array<String>) {
    val (a, b) = readLine()!!.split(" ").map { it.toInt() }

    if (a * b % 2 == 0) {
        println("Even")
    } else {
        println("Odd")
    }
}

File: go/atcoder/tokubeki10mon/01_abc086a.go
package main

import "fmt"

func main() {
	var a, b int
	fmt.Scan(&a, &b)
	if product := a * b; product%2 == 0 {
		fmt.Println("Even")
	} else {
		fmt.Println("Odd")
	}
}
